scripts/list_boards.py

```diff
@@ -219,9 +267,9 @@ def load_v2_boards(board_name, board_yml, systems):
             socs = [Soc.from_soc(systems.get_soc(s['name']), s.get('variants', []))
                     for s in board.get('socs', {})]
 
-            board = Board(
+            boards.update({board['name']: Board(
```

Comment:
Isn't this the same as:
```suggestion
            boards[board['name']] = Board(
```

Reply:
yes, guess I just was having `update` from other places.
Do you prefer a change, or is it simply because the other is shorter ?

Reply:
I think this change would be cleaner, especially compared to having `)})` at the bottom of the statement.

Reply:
fair point, fixed